x-pack/plugins/security_solution/common/detection_engine/constants.ts

```diff
@@ -32,3 +34,6 @@ export const PREBUILT_RULES_PACKAGE_NAME = 'security_detection_engine';
  * Rule signature id (`rule.rule_id`) of the prebuilt "Endpoint Security" rule.
  */
 export const ELASTIC_SECURITY_RULE_ID = '9a1a2dae-0b5f-4c3d-8305-a268d404c306';
+
+export const DEFAULT_SUPPRESSION_MISSING_FIELDS_STRATEGY: AlertSuppressionMissingFields =
```

Comment:
Nit: We could improve cohesion by keeping it close to where the type is defined: in `./rule_schema/model/specific_attributes/query_attributes`:

```ts
/**
 * describes how alerts will be generated for documents with missing suppress by fields
 */
export enum AlertSuppressionMissingFieldsStrategy {
  // per each document a separate alert will be created
  DoNotSuppress = 'doNotSuppress',
  // only alert will be created per suppress by bucket
  Suppress = 'suppress',
}

export const ALERT_SUPPRESSION_MISSING_FIELDS_DEFAULT_STRATEGY = AlertSuppressionMissingFields.Suppress;
```